Taskfile.yaml

```diff
@@ -2,10 +2,17 @@
 
```

Comment:
seems ok for me, but can we explain in the readme that the app now needs task in prerequisites to test/launch/etc ? and maybe give an example, since it's less mainstream than make

Reply:
Yes! Good idea 😃

Reply:
It's done, tell me what you think :)